[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"123456"}
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>登录成功</h1>
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>登录成功</h1>
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"1234567"}
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>失败，用户名或密码错误</h1>
[2019-03-11 09:29:37] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>失败，用户名或密码错误</h1>
[2019-03-11 09:37:19] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 09:37:19] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:37:19] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"123456"}
[2019-03-11 09:37:19] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>登录成功</h1>
[2019-03-11 09:37:19] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>登录成功</h1>
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"123456"}
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>登录成功</h1>
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>登录成功</h1>
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"1234567"}
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>失败，用户名或密码错误</h1>
[2019-03-11 09:37:24] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>失败，用户名或密码错误</h1>
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"123456"}
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>登录成功</h1>
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>登录成功</h1>
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 11] url：http://115.28.108.130:5000/api/user/login/
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"1234567"}
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 13] 期望结果：<h1>失败，用户名或密码错误</h1>
[2019-03-11 09:38:43] INFO [log_case_info: case_log.py, 14] 实际结果：<h1>失败，用户名或密码错误</h1>
[2019-03-11 09:39:26] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 09:39:26] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 09:41:47] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 09:41:47] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 09:42:52] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123456", "username": "admin"}
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 13] 期望结果：{"code": "0000", "expire": 43200, "msg": "\u6210\u529f", "token": "7519eb2a67b58a074bbffb1f208cadc8"}
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "0000", "expire": 43200, "msg": "成功", "token": "7519eb2a67b58a074bbffb1f208cadc8"}
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123451", "username": "admin"}
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 13] 期望结果：{"code": "501", "msg": "\u8d26\u53f7\u6216\u5bc6\u7801\u4e0d\u6b63\u786e"}
[2019-03-11 09:48:32] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "501", "msg": "账号或密码不正确"}
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123456", "username": "admin"}
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 13] 期望结果：{"code": "0000", "expire": 43200, "msg": "\u6210\u529f", "token": "7519eb2a67b58a074bbffb1f208cadc8"}
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "0000", "expire": 43200, "msg": "成功", "token": "7519eb2a67b58a074bbffb1f208cadc8"}
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123451", "username": "admin"}
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 13] 期望结果：{"code": "501", "msg": "\u8d26\u53f7\u6216\u5bc6\u7801\u4e0d\u6b63\u786e"}
[2019-03-11 09:59:45] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "501", "msg": "账号或密码不正确"}
[2019-03-11 10:15:40] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 10:15:40] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 10:15:40] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123451", "username": "admin"}
[2019-03-11 10:15:40] INFO [log_case_info: case_log.py, 13] 期望结果："\u8d26\u53f7\u6216\u5bc6\u7801\u4e0d\u6b63\u786e"
[2019-03-11 10:15:40] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "501", "msg": "账号或密码不正确"}
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_reg_exist
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 12] 请求参数：{"name": "张三","password":"123456"}
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 13] 期望结果：{"code": "100001", "data": {"name": "\u5f20\u4e09", "password": "e10adc3949ba59abbe56e057f20f883e"}, "msg": "\u5931\u8d25\uff0c\u7528\u6237\u5df2\u5b58\u5728"}
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "501", "msg": "账号或密码不正确"}
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_reg_normal
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 12] 请求参数：{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 13] 期望结果：{"code": "100000", "data": {"name": "\u8303\u51b0\u51b0", "password": "e10adc3949ba59abbe56e057f20f883e"}, "msg": "\u6210\u529f"}
[2019-03-11 10:35:01] INFO [log_case_info: case_log.py, 14] 实际结果：{"code": "0000", "expire": 43200, "msg": "成功", "token": "7519eb2a67b58a074bbffb1f208cadc8"}
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_normal
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123456", "username": "admin"}
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 13] 期望结果：成功
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 14] 实际结果：成功
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 10] 测试用例：test_user_login_password_wrong
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 12] 请求参数：{ "password": "123451", "username": "admin"}
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 13] 期望结果：账号或密码不正确
[2019-03-11 10:55:01] INFO [log_case_info: case_log.py, 14] 实际结果：账号或密码不正确
[2019-03-11 15:35:03] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 15:35:40] ERROR [get_test_data: read_excel.py, 23] 用例数据不存在
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 12] 请求参数：{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 13] 请求头:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 14] 请求体:成功
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 15] 期望结果：<Response [200]>
[2019-03-11 16:52:04] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 12] 请求参数：{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 13] 请求头:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 14] 请求体:成功
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 15] 期望结果：<Response [200]>
[2019-03-11 16:58:09] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 12] 请求参数：{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 13] 请求头:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 14] 请求体:成功
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 15] 期望结果：<Response [200]>
[2019-03-11 16:59:15] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 14] 请求体:成功
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 15] 期望结果：<Response [200]>
[2019-03-11 17:05:10] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:05:52] INFO [log_case_info: case_log.py, 16] 实际结果：<Response [200]>
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:06:59] INFO [log_case_info: case_log.py, 16] 实际结果：<Response [200]>
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:09:19] INFO [log_case_info: case_log.py, 16] 实际结果：<Response [200]>
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:12:05] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:14:06] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:15:35] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:16:19] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:18:52] INFO [log_case_info: case_log.py, 16] 实际结果：JSON parse error: Unrecognized token 'password': was expecting ('true', 'false' or 'null'); nested exception is com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'password': was expecting ('true', 'false' or 'null')
 at [Source: (PushbackInputStream); line: 1, column: 10]
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:19:35] INFO [log_case_info: case_log.py, 16] 实际结果：JSON parse error: Unrecognized token 'password': was expecting ('true', 'false' or 'null'); nested exception is com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'password': was expecting ('true', 'false' or 'null')
 at [Source: (PushbackInputStream); line: 1, column: 10]
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:19:51] INFO [log_case_info: case_log.py, 16] 实际结果：JSON parse error: Cannot construct instance of `com.xw.web.admin.form.SysLoginForm` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('{
  "password": "123456",
  "username": "admin"
}'); nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `com.xw.web.admin.form.SysLoginForm` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('{
  "password": "123456",
  "username": "admin"
}')
 at [Source: (PushbackInputStream); line: 1, column: 1]
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:20:52] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 15] 期望结果：账号或密码不正确
[2019-03-11 17:21:17] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:21:57] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:22:14] INFO [log_case_info: case_log.py, 16] 实际结果：JSON parse error: Unrecognized token 'password': was expecting ('true', 'false' or 'null'); nested exception is com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'password': was expecting ('true', 'false' or 'null')
 at [Source: (PushbackInputStream); line: 1, column: 10]
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:27:15] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:28:04] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:30:00] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:31:38] INFO [log_case_info: case_log.py, 16] 实际结果：JSON parse error: Cannot construct instance of `com.xw.web.admin.form.SysLoginForm` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('{
  "password": "123456",
  "username": "admin"
}'); nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `com.xw.web.admin.form.SysLoginForm` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('{
  "password": "123456",
  "username": "admin"
}')
 at [Source: (PushbackInputStream); line: 1, column: 1]
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:44:58] INFO [log_case_info: case_log.py, 16] 实际结果：JSON parse error: Cannot construct instance of `com.xw.web.admin.form.SysLoginForm` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('"{\n  \"password\": \"123456\",\n  \"username\": \"admin\"\n}"'); nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `com.xw.web.admin.form.SysLoginForm` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('"{\n  \"password\": \"123456\",\n  \"username\": \"admin\"\n}"')
 at [Source: (PushbackInputStream); line: 1, column: 1]
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:45:15] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:45:58] INFO [log_case_info: case_log.py, 16] 实际结果：账号或密码不正确
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 17:49:06] INFO [log_case_info: case_log.py, 16] 实际结果：成功
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 10] 测试用例：登录接口
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 11] url：http://192.168.1.171:8083/sys/login
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 12] 请求参数：None
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 13] 请求头:{"Content-Type":"application/json;charset=UTF-8"}
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 14] 请求体:{
  "password": "123456",
  "username": "admin"
}
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 15] 期望结果：成功
[2019-03-11 18:04:14] INFO [log_case_info: case_log.py, 16] 实际结果：成功
